file_metadata/image/svg_file.py

```diff
@@ -35,3 +35,15 @@ def fetch(self, key=''):
                 return name
 
         return super(SVGFile, self).fetch(key)
+
+    def analyze_file_format(self):
+        """
+        Simply add a metadata mentioning this is a valid SVG file. This is
+        useful because SVG cannot be simply detected by MimeType as it's not
+        standardized yet.
```

Comment:
I do have a SVG checking method - not using py_w3c, but an equivalent one. That's how this class is decided upon by GenericFile